lib/ansible/modules/messaging/rabbitmq_user.py

```diff
@@ -31,8 +31,8 @@
   password:
     description:
       - Password of user to add.
-      - To change the password of an existing user, you must also specify
-        C(force=yes).
+      - To change the password of an existing user for RabbitMQ < 3.6.5,
+        you must also specify C(force=yes).
```

Comment:
Could you rephrase this a bit to show users this feature is available since 2.4.